pkg/systemd/overview-cards/usageCard.jsx

```diff
@@ -29,59 +29,91 @@ import "./usageCard.less";
 
 const _ = cockpit.gettext;
 
-const UPDATE_DELAY = 5000;
+const METRICS_SPEC = {
+    payload: "metrics1",
+    source: "internal",
+    interval: 5000,
```

Comment:
Subjectiveky, 5 seconds feels slow.  `top` has a 3 second interval by default, maybe we should use the same?

Reply:
Agreed, moved to 3s.